config.yaml

```diff
@@ -2,20 +2,32 @@ options:
   core-interface:
     type: string
     description: Host interface to use for the Core Network.
+  core-interface-mtu:
+    type: int
+    default: 1500
+    description: MTU to use for the Core Network interface
   core-gateway-ip:
     type: string
     default: 192.168.250.1/24
     description: Gateway IP address to the Core Network.
   access-interface:
     type: string
     description: Host interface to use for the Access Network.
+  access-interface-mtu:
+    type: int
+    default: 1500
```

Comment:
I am not sure how Multus decides this value and could it vary depend on a specification of environment. My opinion is not to add any MTU config if it's not set by the user.

Reply:
The problem is we have no method of documenting what the MTU will be if it is not specified.  The multus docs just say "the default from the kernel," but there is no such thing.  1500 is the accepted standard and again, let's compare this to the UPF change - which one makes more sense and we should go with that,